Backend/src/NHSD.BuyingCatalogue.Identity.Api/Infrastructure/ReturnUrlParser.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
```

Comment:
Can we bring this file in when we actually need it? Doesn't look like it is currently been used.